packages/pulse-next/package.json

```diff
@@ -25,5 +25,8 @@
     "type": "git",
     "url": "https://github.com/pulse-framework/pulse"
   },
-  "gitHead": "df2d02d8e5a07ded7bb792c0d8e1473c5b6159ba"
+  "gitHead": "df2d02d8e5a07ded7bb792c0d8e1473c5b6159ba",
+  "files": [
+    "dist/**/*"
```

Comment:
As far as I'm aware, `files` is supposed to be an exhaustive list of input files — `include` should be used for any nonspecific paths (such as the recursive glob pattern here). I'll have to run this by when I test.

Reply:
I don't know what the advantage of 'include' is I only know that it worked with 'files' for agileTs quite fine..

Reply:
* Edit.. Idk if include workes at all.. 
I tested it with ` "include ": "dist/**/*"` and it didn't exclude the tests and lib

This is the link how I got to 'files':
https://remarkablemark.org/blog/2017/09/23/npm-publish-include-files/

Reply:
Ahh, I confused package.json for tsconfig.json! I'm sorry!